hibike/hibikeDevices.json

```diff
@@ -1,288 +1,83 @@
-                        "machineName": "value0",
-                        "humanName": "Red"
-                    },
-                    {
-                        "scalingFactor": 1.0,
-                        "machineName": "value1",
-                        "humanName": "Green"
-                    },
-                    {
-                        "scalingFactor": 1.0,
-                        "machineName": "value2",
-                        "humanName": "Blue"
-                    },
-                    {
-                        "scalingFactor": 1.0,
-                        "machineName": "value3",
-                        "humanName": "Clear"
-                    }
-                ]
-            },
-            "params": [
-                "dataUpdate",
-                "IntegrationTime",
-                "Gain",
-                "Toggle"
-            ]
-        },
-        {
-            "deviceID": "0x0011",
-            "deviceName": "MetalDetector",
-            "dataFormat": {
-                "formatString": "<i",
-                "parameters": [
-                    {
-                        "scalingFactor": 1.0,
-                        "machineName": "value0",
-                        "humanName": "Reading"
-                    }
-                ]
-            },
-            "params": [
-                "dataUpdate",
-                "calibrate"
-            ]
-        },
-        {
-            "deviceID": "0xFFFF",
-            "deviceName": "ExampleDevice",
-            "dataFormat": {
-                "formatString": "<?BHIQ",
-                "parameters": [
-                    {
-                        "scalingFactor": 1.0,
-                        "machineName": "value0",
-                        "humanName": "Example 0"
-                    },
-                    {
-                        "scalingFactor": 1.0,
-                        "machineName": "value1",
-                        "humanName": "Example 1"
-                    },
-                    {
-                        "scalingFactor": 2.0,
-                        "machineName": "value2",
-                        "humanName": "Example 2"
-                    },
-                    {
-                        "scalingFactor": 3.0,
-                        "machineName": "value3",
-                        "humanName": "Example 3"
-                    },
-                    {
-                        "scalingFactor": 0.25,
-                        "machineName": "value4",
-                        "humanName": "Example 4"
-                    }
-                ]
-            },
-            "params": [
-                "dataUpdate",
-                "test0",
-                "test1",
-                "test2",
-                "test3",
-                "test4",
-                "test5"
-            ]
-        }
-    ]
-}
\ No newline at end of file
+[
+    {"id": 0,     "name": "LimitSwitch",   "params": [    
```

Comment:
BTW, the formatting here is super awkward in diff view. Is there any change you could adjust the whitespace in this file?